pkgs/build-support/fetchnextcloudapp/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, pkgs, ... }:
+{ name
+, url
+, version
+, sha256
+, patches ? [ ]
+}:
+stdenv.mkDerivation {
+  name = "nc-app-${name}";
+  version = "${version}";
+
+  src = pkgs.fetchurl {
+    url = url;
+    sha256 = sha256;
```

Comment:
I still think that these `foo = foo` expressions are kinda weird. Instead you'd generally use `inherit foo`.

Reply:
Yep, we prefer to use `inherit`.